src/XML/ec/InclusiveNamespaces.php

```diff
@@ -62,7 +62,7 @@ public static function fromXML(DOMElement $xml): object
     {
         $prefixes = self::getAttribute($xml, 'PrefixList', '');
 
-        return new self(explode(' ', $prefixes));
+        return new self(array_filter(explode(' ', $prefixes), function ($v, $k) { return !empty($v); }));
```

Comment:
From the PHP-docs:   "If no callback is supplied, all empty entries of array will be removed. See empty() for how PHP defines empty in this case."

Reply:
Good catch 😉 

I thought about leaving it off as well, but on a second thought, explicit is better, since then we know it will work even if for whatever reason `' '` stops being evaluated to `false` in the future.